### PR TITLE
Rails Milstone 

### PR DESCRIPTION
**Pull request by Tchilo**
## The following was done:
- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize  project with Git.
- Make  project with a Postgres database set up.
- Create a table for storing  messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table .

### Link to react pullrequest [Click Me!](https://github.com/Tchilo/hello-react-front-end/pull/1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
+Contributions, issues, and feature requests are welcome!
 
-* Services (job queues, cache servers, search engines, etc.)
+Feel free to check the [issues page](../../issues/).
 
-* Deployment instructions
+## Show your support
 
-* ...
+Give a ⭐️ if you like this project!
```
Comment (t-yanick) [inline README.md:36]: - [ ]  Please in order to make your `Readme file` more informative and professional, kindly follow this [template](https://github.com/microverseinc/readme-template) from Microverse and adapt it to your project. Documenting your project is a great skill to have as a developer. The readme of your backend is missing much vital information, you did a great job with that of the front-end.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Tchilo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Tchilo) [reply README.md:36]: thanks for the code review 😄

Comment (Whoistolu) [reply README.md:36]: Well resolved @Tchilo 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Tchilo 

Great job implementing the recommended changes 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**